data/src/main/java/com/microsoft/azure/kusto/data/exceptions/DataWebException.java

```diff
@@ -9,7 +9,7 @@
 public class DataWebException extends Exception{
 
     private String message;
```

Comment:
This property is unnecessary, as its parent class, Exception, already has this. Just call super(message) in the constructors below.